specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/Registration.json

```diff
@@ -171,6 +171,49 @@
                         }
                     }
                 }
+            },
+			"patch": {
+                "x-ms-examples": {
+                    "Create or update an Azure Stack registration.": {
+                        "$ref": "examples/Registration/Put.json"
```

Comment:
@AndyTao0402 - looks like it's referencing the wrong example?

Reply:
True. Fixed as well as the description message in above line.